From a243de941e2c90fd8d53c4980bf483253d8726ef Mon Sep 17 00:00:00 2001
From: Abhishek <apatgaonkar@habana.ai>
Date: Mon, 06 Dec 2021 08:08:16 +0200
Subject: [PATCH] [SW-65065] preboot: change function name

This patch changes function name from hl_is_btl_flow
to hl_is_legacy_boot which is more clear and
precise.

Change-Id: I4c52a544cae0d6553dc5d52a708365e351a641a7
---

diff --git a/app/preboot/preboot.c b/app/preboot/preboot.c
index cbf3a24..5912446 100644
--- a/app/preboot/preboot.c
+++ b/app/preboot/preboot.c
@@ -133,7 +133,7 @@
 	return preboot_vers[ver];
 }
 
-__weak bool hl_is_btl_flow(void)
+__weak bool hl_is_legacy_boot(void)
 {
 	return true;
 }
@@ -145,7 +145,7 @@
 	int i, btl_ver_sz;
 
 	preboot_puts("\n");
-	if (hl_is_btl_flow()) {
+	if (hl_is_legacy_boot()) {
 		preboot_puts_raw(preboot_vers[PREBOOT_BTL_VER]);
 		preboot_puth(btlver, true);
 	}
diff --git a/arch/gaudi/arm/common.c b/arch/gaudi/arm/common.c
index 947cb98..571d007 100644
--- a/arch/gaudi/arm/common.c
+++ b/arch/gaudi/arm/common.c
@@ -640,7 +640,7 @@
 	return HL_BOOT_INFO_VERSION;
 }
 
-bool hl_is_btl_flow(void)
+bool hl_is_legacy_boot(void)
 {
 	u32 boot_strap = regmap_readl(mmPSOC_GLOBAL_CONF_BOOT_STRAP_PINS);
 
diff --git a/arch/gaudi2/common/common.c b/arch/gaudi2/common/common.c
index 85ff6c3..eca8b59 100644
--- a/arch/gaudi2/common/common.c
+++ b/arch/gaudi2/common/common.c
@@ -1017,7 +1017,7 @@
 	}
 }
 
-bool hl_is_btl_flow(void)
+bool hl_is_legacy_boot(void)
 {
 	u32 boot_strap = regmap_readl(mmPSOC_GLOBAL_CONF_BOOT_STRAP_PINS_L);
 
diff --git a/arch/goya/arm/common.c b/arch/goya/arm/common.c
index aec8c83..c8a4ca4 100644
--- a/arch/goya/arm/common.c
+++ b/arch/goya/arm/common.c
@@ -297,7 +297,7 @@
 	return PCI_IDS_GOYA;
 }
 
-bool hl_is_btl_flow(void)
+bool hl_is_legacy_boot(void)
 {
 	u32 boot_strap = regmap_readl(mmPSOC_GLOBAL_CONF_BOOT_STRAP_PINS);
 
diff --git a/arch/greco/arc/common.c b/arch/greco/arc/common.c
index 8da3d5a..9617a94 100644
--- a/arch/greco/arc/common.c
+++ b/arch/greco/arc/common.c
@@ -151,7 +151,7 @@
 	regmap_writel(ARC_RST_VALUE, mmPSOC_ARC0_CFG_ARC_RST);
 }
 
-bool hl_is_btl_flow(void)
+bool hl_is_legacy_boot(void)
 {
 	/*
 	 * Due to DCCM ECC issue in arcboot rom, hybrid mode is enabled
